src/OGL3X/Shaders_ogl3x.h

```diff
@@ -150,12 +150,14 @@ MAIN_SHADER_VERSION
 
 static const char* fragment_shader_header_common_variables =
 MAIN_SHADER_VERSION
+#ifndef GLESX
```

Comment:
Bad idea. These are GL ES extensions. You will kill mip-mapping emulation on mobile devices.